hadoop-ozone/dist/src/main/smoketest/security/ozone-secure-tenant.robot

```diff
@@ -66,6 +66,13 @@ Verify Bucket 1 Owner
     ${result} =         Execute          ozone sh bucket info /tenantone/bucket-test1 | jq -r '.owner'
                         Should Be Equal  ${result}       testuser
 
+Put, get and delete a key in the tennant bucket
```

Comment:
```suggestion
Put, get and delete a key in the tenant bucket
```

Reply:
Seems like a basic test that should have been present :-) Thanks for adding this.